django/db/models/options.py

```diff
@@ -65,9 +65,9 @@ def make_immutable_fields_list(name, data):
 
 @python_2_unicode_compatible
 class Options(object):
-    FORWARD_PROPERTIES = ('fields', 'many_to_many', 'concrete_fields',
-                          'local_concrete_fields', '_forward_fields_map')
-    REVERSE_PROPERTIES = ('related_objects', 'fields_map', '_relation_tree')
+    FORWARD_PROPERTIES = {'fields', 'many_to_many', 'concrete_fields',
+                          'local_concrete_fields', '_forward_fields_map'}
+    REVERSE_PROPERTIES = {'related_objects', 'fields_map', '_relation_tree'}
```

Comment:
While my tests suggest that iterating a set is ~8% slower in this case, it is a negligible difference once you look at the whole `_expire_cache` function.